### ISSUE TITLE
CE-355 Stop and Go elements on Projects page are announced as images by the screen reader

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7208]
### Where To Find Bug
`/projects`

### Expected Behaviour
The screen reader should announce the stop and go elements as buttons

### Actual Behaviour
The screen reader announces the stop and go elements as images

### Steps To Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Turn-on your screen reader of choice
- Sign in to Scratch
- Click on a project
- Tab to the green go icon and the red stop icon
- Notice that the screen reader refers to these as images

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)
